tests/materialize/materialize-redshift/snapshot.json

```diff
@@ -645,6 +645,42 @@
       "num_and_str": 5.1,
       "num_str": 50.1,
       "time": "23:59:59Z"
+    },
+    {
+      "date": null,
+      "datetime": null,
+      "flow_document": "{\"_meta\":{\"uuid\":\"c02bd79a-20e1-11ee-990b-ffd12dfcd47f\"},\"id\":8,\"num_str\":\"NaN\"}",
+      "flow_published_at": "2023-07-12 18:27:25.889321 +0000 UTC",
+      "id": 8,
+      "int_and_str": null,
+      "int_str": null,
+      "num_and_str": null,
+      "num_str": null,
+      "time": null
+    },
+    {
+      "date": null,
+      "datetime": null,
+      "flow_document": "{\"_meta\":{\"uuid\":\"c02bd79a-20e1-11ee-990b-ffd12dfcd47f\"},\"id\":9,\"num_str\":\"Infinity\"}",
+      "flow_published_at": "2023-07-12 18:27:25.889321 +0000 UTC",
+      "id": 9,
+      "int_and_str": null,
+      "int_str": null,
+      "num_and_str": null,
+      "num_str": null,
+      "time": null
+    },
+    {
+      "date": null,
+      "datetime": null,
+      "flow_document": "{\"_meta\":{\"uuid\":\"c02bd79a-20e1-11ee-990b-ffd12dfcd47f\"},\"id\":10,\"num_str\":\"-Infinity\"}",
+      "flow_published_at": "2023-07-12 18:27:25.889321 +0000 UTC",
+      "id": 10,
+      "int_and_str": null,
+      "int_str": null,
+      "num_and_str": null,
+      "num_str": null,
+      "time": null
```

Comment:
How did we end up with these new records in the snapshot? 🤔

Reply:
From #1037, which added some new test cases, and must not have run the redshift test/updated the snapshots. I ran it for testing this PR and updated the snapshot, which brought in that change as well, although unrelated to anything here.